pkg/features/scheduler/scheduler.go

```diff
@@ -0,0 +1,24 @@
+package scheduler
```

Comment:
I suggest renaming the file to `pkg/features/features.go`.

Reply:
Sounds reasonable, I have updated.